pkg/ccl/changefeedccl/sink_webhook_v2.go

```diff
@@ -209,7 +236,32 @@ func (sc *webhookSinkClient) FlushResolvedPayload(
 	})
 }
 
-// Flush implements the SinkClient interface
+// readResponseBody handles response body reading and decompression if needed.
+func (sc *webhookSinkClient) readResponseBody(res *http.Response) ([]byte, error) {
+	defer res.Body.Close()
```

Comment:
it looks like this body is already defer-closed on line 276

Reply:
Fixed